emsdk_manifest.json

```diff
@@ -633,6 +633,7 @@
     "bitness": 64,
     "url": "https://github.com/juj/ccache.git",
     "git_branch": "emscripten",
+    "activated_cfg": "CCACHE=%installation_dir%/bin",
     "activated_path": "%installation_dir%/bin",
```

Comment:
This line should add ccache to your PATH.  You would need to be sure to include it in your `emsdk activate` command line though I think.

Reply:
Without `activated_cfg`, this clause causes ccache to not be added to the path:

https://github.com/emscripten-core/emsdk/blob/c18280c8f3f8f088b4337f0f6a5fdb2be0d67179/emsdk.py#L1733-L1735

Reply:
`activated_cfg` doesn't have anything to do with the PATH.   `activated_cfg` is used to add things to the emscripten config file.  As far as I know ccache has not place in the emscripten config file.

The things that adds to your PATH is `activated_path`.

Reply:
Okay, then it would seem we need to change L1733-L1735 here, as this results in any tool without an activated_cfg and no deps being excluded. ccache is the only tool without an activated_cfg as far as I can tell.

Reply:
I _think_ if we change L1735 to `return True` then this will result in ccache always being active if it's installed? I'm not super familiar with how emsdk works—where is the list of currently activated tools stored?

Reply:
Yes, it looks like we can completely remove those lines I think.

I'm not sure why would declare a tool inactive simply because it has no dependencies.

Reply:
@juj any idea why these lines exist?